data_filter_azure/example_documentdb.rego

```diff
@@ -0,0 +1,47 @@
+package documentdb.example
+
+# data.example.allow == true
+#
+# {"input": {"method": "GET", "path": ...}, "unknowns": ["data.posts"]}
+
+default allow = false
+
+allow {
+    allowed[_]
+}
+
+allowed[user] {
+    data.permissions[user]
+    user.registry == input.registry
+    user.user == input.user
+    user.map[_].type == input.type
+    user.map[_].name == input.resourceName
+    contains(user.map[_].actions, input.action)
```

Comment:
If 'data.permissions' is unknown, then none of these rules or even statements within the rules get evaluated, and we're just translating this bunch of rules into a SQL query.  Is that what we intend?

Reply:
In DocumentDb scenario, permissions is unknown and we partially evaluate to get the queries that can be translated to SQL. We then rely on the SQL store (in this case Document DB) to actually evaluate for the decision. Please correct me if my understanding is wrong.